src/Node/Buffer/Internal.js

```diff
@@ -38,7 +37,7 @@ exports.writeStringInternal = function (encoding) {
 exports.setAtOffset = function (value) {
   return function (offset) {
     return function (buff) {
-      return function() {
+      return function () {
         buff[offset] = value;
         return {};
```

Comment:
And here again

Reply:
That's right. Leaving it for a later PR, though!